source/deployment/push.rst

```diff
@@ -64,8 +64,9 @@ To deploy in production with privately-hosted mobile apps compiled by your organ
 1. Compile your own iOS, Android mobile applications and Mattermost Push Notification Service (MPNS) via their open source repositories:
 
   - `Open source repository for the Mattermost Push Notification Service <https://github.com/mattermost/push-proxy>`_
-  - `Open source repository for the Mattermost iOS application <https://github.com/mattermost/ios>`_
-  - `Open source repository for the Mattermost Android application <https://github.com/mattermost/android>`_
+  - `Open source repository for the Mattermost iOS and Android applications <https://github.com/mattermost/mattermost-mobile>`_
+  - `Open source repository for the Mattermost Classic iOS application <https://github.com/mattermost/ios>`_
+  - `Open source repository for the Mattermost Classic Android application <https://github.com/mattermost/android>`_
```

Comment:
I think these repos have been renamed and are now https://github.com/mattermost/mattermost-ios-classic and https://github.com/mattermost/mattermost-android-classic